BusBoard.ConsoleApp/Helpers.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using RestSharp;
+using RestSharp.Serialization.Json;
+
+namespace BusBoard.ConsoleApp
+{
+    class Helpers
```

Comment:
Can you think of a more descriptive name? Maybe name it `ApiHelper` and also make it responsible for creating the client? If it were a real application I would suggest making the methods non-static and making the client a private variable so that you can reuse it. As this is a fairly small thing though, it's up to you 🙂